library/sysrc

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
```

Comment:
No this isn't deployed to the server, is an ansible module to allow it to integrate with the `sysrc` command that is present in the FreeBSD base system.